release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1289,6 +1289,40 @@ Any registry on this list is not required to have an entry in the pull secret us
 
 For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster.adoc#ztp-configuring-the-hub-cluster-to-use-unauthenticated-registries_ztp-preparing-the-hub-cluster[Configuring the hub cluster to use unauthenticated registries].
 
+[id="ocp-4-12-0-ironic-agent-image-ztp"]
+==== Ironic agent mirroring in disconnected GitOps ZTP installations
+
+For disconnected installations using GitOps ZTP, if you are deploying {product-title} version 4.11 or earlier to a spoke cluster with converged flow enabled, you must mirror the default Ironic agent image to the local image repository. The default Ironic agent images are the following:
+
+* AMD64 Ironic agent image: `quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:d3f1d4d3cd5fbcf1b9249dd71d01be4b901d337fdc5f8f66569eb71df4d9d446`
+
+* AArch64 Ironic agent image: `quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:cb0edf19fffc17f542a7efae76939b1e9757dc75782d4727fb0aa77ed5809b43`
+
+For more information about mirroring images, see xref:../installing/disconnected_install/installing-mirroring-installation-images.adoc#installation-mirror-repository_installing-mirroring-installation-images[Mirroring the OpenShift Container Platform image repository].
+
+[id="support-for-dual-stack-addressing-for-the-API-VIP-and-Ingress-VIP"]
+= Support for dual-stack addressing for the API VIP and Ingress VIP
```

Comment:
Checking that this is the right heading level - not sure if it's supposed to go under "new features and enhancements" in which case it would be `==`, I think? (I think the guideline is only one H1 per adoc file is allowed.)

Reply:
Moved content and updated to H4.